src/core/Alert/Alert.tsx

```diff
@@ -20,7 +20,11 @@ export type AlertProps = {
    */
   clickableText?: React.ReactNode;
   /**
-   * Action handler for the clickable text.
+   * Props for the clickable text. Used for providing `href` and other attributes.
+   */
+  clickableTextProps?: React.ComponentPropsWithoutRef<'a'>;
+  /**
+   * Action handler for the clickable text, used when href is not provided.
```

Comment:
I dont think adding comment `when href is not provided` is actually right :D

Reply:
Added deprecation